### PR TITLE
feat: Updating Trade section in README

### PR DESCRIPTION
**Pull request by erdimaden**
### What changed? Why?
Updating Trade section in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,6 +148,23 @@ const anotherWallet = await user.createWallet();
...
+
+### Trading Funds
+
+```typescript
+// Create a wallet on `base-mainnet` to trade assets with.
+let mainnetWallet = await user.createWallet({ networkId: Coinbase.networks.BaseMainnet });
+
+console.log(`Wallet successfully created: ${mainnetWallet}`);
+
+// Fund wallet's default address with ETH from an external source.
```
Comment (John-peterson-coinbase) [inline README.md:160]:
```suggestion
// Fund your Wallet's default Address with ETH from an external source.
```

File: README.md [hunk 2]
```diff
@@ -148,6 +148,23 @@ const anotherWallet = await user.createWallet();
 const transfer = await wallet.createTransfer(0.00001, Coinbase.assets.Eth, anotherWallet);
 ```
 
+
+### Trading Funds
+
+```typescript
+// Create a wallet on `base-mainnet` to trade assets with.
```
Comment (John-peterson-coinbase) [inline README.md:155]:
```suggestion
// Create a Wallet on `base-mainnet` to trade assets with.
```

File: README.md (hunk 1 above)
Comment (erdimaden) [reply README.md]: cc: @jazz-cb

File: README.md (hunk 2 above)
Comment (erdimaden) [reply README.md]: cc: @jazz-cb

**Pull request merged**